pkg/tle/tle.go

```diff
@@ -78,7 +78,7 @@ func GenerateTransparencyLogEntry(anon models.LogEntryAnon) (*rekor_pb.Transpare
 			SignedEntryTimestamp: anon.Verification.SignedEntryTimestamp,
 		},
 		InclusionProof: &rekor_pb.InclusionProof{
-			LogIndex: *anon.LogIndex,
+			LogIndex: *anon.Verification.InclusionProof.LogIndex,
```

Comment:
can you please add a comment to denote that the inclusion proof is relative to the specific shard, and therefore the value here might not match the global index on line 68